crt_portal/cts_forms/forms.py

```diff
@@ -256,6 +257,7 @@ def __init__(self, *args, **kwargs):
         self.fields['violation_summary'].help_text = SUMMARY_HELPTEXT
         self.fields['violation_summary'].error_messages = {'required': VIOLATION_SUMMARY_ERROR}
         self.fields['violation_summary'].required = True
+    #    self.fields['language'] = CharField()
```

Comment:
This can be deleted too

Reply:
removed the debug print line.  Added more descriptive text instead of language code.